front-packages/akeneo-design-system/src/components/MainNavigationItem/MainNavigationItem.tsx

```diff
@@ -0,0 +1,149 @@
+import React, {Ref, ReactElement, SyntheticEvent} from 'react';
+import styled from 'styled-components';
+import {AkeneoThemedProps, getColor, getFontSize} from '../../theme';
+import {IconProps} from '../../icons';
+import {Tag} from '../Tags/Tags';
+
+const Container = styled.button<{active: boolean; disabled: boolean} & AkeneoThemedProps>`
+  width: 80px;
+  height: 70px;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  background-color: transparent;
+  outline-style: none;
+  text-decoration: none;
+  line-height: 1.15;
+  white-space: nowrap;
+  cursor: ${({disabled}) => (disabled ? 'not-allowed' : 'pointer')};
+  border: none;
+  border-left: 4px solid
+    ${({active, disabled}) => {
+      return !disabled && active ? getColor('brand', 100) : 'transparent';
+    }};
+
+  color: ${({active, disabled}) => {
+    return disabled ? getColor('grey', 100) : active ? getColor('brand', 100) : getColor('grey', 120);
+  }};
+  svg {
+    color: ${({active, disabled}) => {
+      return disabled ? getColor('grey', 80) : active ? getColor('brand', 100) : getColor('grey', 100);
+    }};
+  }
+
+  :hover {
+    border-color: ${({disabled}) => !disabled && getColor('brand', 100)};
+    color: ${({disabled}) => !disabled && getColor('brand', 100)};
+    svg {
+      color: ${({disabled}) => !disabled && getColor('brand', 100)};
+    }
+  }
+
+  :focus:not(:active) {
+    box-shadow: 0 0 0 2px ${getColor('blue', 40)};
+    outline: none;
+  }
+`;
+
+const Title = styled.p`
+  font-size: ${getFontSize('small')};
+  margin: 0;
+  margin-top: 7px;
+`;
+
+const IconContainer = styled.div`
+  position: relative;
+`;
+
+const CustomTag = styled(Tag)`
```

Comment:
```suggestion
const PositionedTag = styled(Tag)`
```
what do you think of this ? `Custom` is not clear about what your component changes